fe/fe-core/src/main/java/com/starrocks/mysql/privilege/UserProperty.java

```diff
@@ -391,9 +331,6 @@ public void write(DataOutput out) throws IOException {
         Text.writeString(out, qualifiedUser);
         out.writeLong(maxConn);
 
-        // user resource
-        resource.write(out);
```

Comment:
Should call UserResource.write() to make FE can rollback.
otherwise old version can not recognize new-format meta.

Reply:
Good advice. I've changed it to write two dummy fields `num_resources` and `num_groups` to make it compatible.